source/wp-content/themes/wporg-parent-2021/inc/block-styles.php

```diff
@@ -87,6 +87,22 @@ function setup_block_styles() {
 		)
 	);
 
+	register_block_style(
+		'core/list',
```

Comment:
At this stage I'm thinking I'll be adding more style variations to other blocks like query loop, as this style is pretty widespread in the designs, and it doesn't seem like we can reasonably cover all with one implementation.